sqs_listener/__init__.py

```diff
@@ -48,18 +48,20 @@ def __init__(self, queue, **kwargs):
         self._attribute_names = kwargs['attribute_names'] if 'attribute_names' in kwargs else []
         self._force_delete = kwargs['force_delete'] if 'force_delete' in kwargs else False
         self._region_name = kwargs['region_name'] if 'region_name' in kwargs else 'us-east-1'
+        self._endpoint_name = kwargs['endpoint_name'] if 'endpoint_name' in kwargs else None
         self._wait_time = kwargs['wait_time'] if 'wait_time' in kwargs else 0
         # must come last
         self._client = self._initialize_client()
 
 
     def _initialize_client(self):
-        #sqs = boto3.client('sqs', region_name=self._region_name)
-
         # new session for each instantiation
         self._session = boto3.session.Session()
-        sqs = self._session.client('sqs', region_name=self._region_name)
+        ssl = True
+        if self._region_name == 'elasticmq':
```

Comment:
What is the significance of `elasticmq` as a region name?  Are there other reserved names the script should be aware of?

Reply:
@jegesh This is a common pattern when using https://github.com/adamw/elasticmq for testing. `elasticmq` is its default region and will never be a real AWS region, so this should be safe.